bin.js

```diff
@@ -52,7 +52,7 @@ args
   .example('cat log | pino-pretty -m fooMessage', 'To highlight a string at a key other than \'msg\', use')
   .example('cat log | pino-pretty --levelKey fooLevel', 'To detect the log level at a key other than \'level\', use')
   .example('cat log | pino-pretty --levelLabel fooLabel', 'To output the log level label using a key other than \'levelLabel\'')
-  .example('cat log | pino-pretty -a fooTimestamp', 'To display timestamp from a key other than \'time\'')
+  .example('cat log | pino-pretty -a fooTimestamp', 'To display timestamp from a key other than \'time\', use')
```

Comment:
Is there meant to be something after "use"?

Reply:
Yes, 
if you run `-h` it outputs an example after that.
Not all examples/options have the `, use`, i could either add `, use` to all or remove the existing ones, whichever you prefer.

```sh
$ pino-pretty -h
...
    - To display timestamp from a key other than 'time', use
    $ cat log | pino-pretty -a fooTimestamp
...
```

Reply:
Nix the `, use`.

Reply:
Done